src/ray/object_manager/pull_manager.cc

```diff
@@ -740,6 +741,21 @@ int64_t PullManager::NextRequestBundleSize(const Queue &bundles,
   return bytes_needed_calculated;
 }
 
+void PullManager::RecordMetrics() const {
+  absl::MutexLock lock(&active_objects_mu_);
+  ray::stats::STATS_num_bytes_available.Record(num_bytes_available_);
+  ray::stats::STATS_num_bytes_being_pulled.Record(num_bytes_being_pulled_);
+  ray::stats::STATS_num_bytes_being_pulled_pinned.Record(pinned_objects_size_);
```

Comment:
Should use labels for breakdown. Also, please prefix metrics with compoment, like `STATS_pull_manager_`